src/app/frank/fd_frank_quic.c

```diff
@@ -133,13 +140,14 @@ static long allow_syscalls[] = {
   __NR_nanosleep, /* fd_tempo_tick_per_ns calibration */
   __NR_getpid,    /* OpenSSL RAND_bytes checks pid, temporarily used as part of quic_init to generate a certificate */
   __NR_getrandom, /* OpenSSL RAND_bytes reads getrandom, temporarily used as part of quic_init to generate a certificate */
+  __NR_sendto,    /* fd_xsk requires sendto */
```

Comment:
This should be changed to __NR_send to be maximally restrictive. I'm not sure why that wasn't defined for you, but maybe it's not in syscalls.h and you need another header. The syscall definitely exists.

Reply:
__NR_send isn't available on our current hosts